bin/cocos.py

```diff
@@ -393,6 +393,10 @@ def parse_args(self, argv):
         parser.add_argument("-p", "--platform",
                             dest="platform",
                             help="select a platform (%s)" % ', '.join(platform_list))
+        parser.add_argument("--proj-dir",
```

Comment:
In fact, the parameter `--proj-dir` is used for `cocos compile`. So, it should be added into the `project_compile.py`.